@stellar/anchor-tests/src/tests/sep31/transactions.ts

```diff
@@ -97,6 +112,97 @@ const canCreateTransaction: Test = {
       // but to satisfy TypeScript we make these checks here.
       throw "improperly configured";
     }
+
+    async function pollCustomerAccepted(
+      getCustomerNetworkCall: NetworkCall,
+      timeout: number,
+    ) {
+      for (let i = 0; i < timeout; i++) {
+        const resBody = await makeRequest(
+          getCustomerNetworkCall,
+          200,
+          result,
+          "application/json",
+        );
+        if (resBody.status === "ACCEPTED") {
+          return;
+        }
+        await new Promise((resolve) => setTimeout(resolve, 1000));
+      }
```

Comment:
If I configure `customerPollingTimeout: 30`, then this code would make 30 requests before throwing an error.

Instead, lets treat `customerPollingTimeout` as the number of seconds the test should wait before throwing the error.

```js
let startedAt = new Date.now();
while(startedAt + timeout > new Date.now()) {
    ...
    await new Promise((resolve) => setTimeout(resolve, 1000));
}
```